src/client/styles/cards.css

```diff
@@ -123,10 +123,10 @@
 }
 
 .playingCards {
-    display: table;
     width: 100%;
     padding-top: 20px;
     height: 250px;
+    overflow: auto;
```

Comment:
These CSS changes somehow change the z-level of the cards. On hover, the card now hides behind the rest of the game and is not visible anymore...

Reply:
I've now just got rid of this and set a card limit